### ISSUE TITLE
JSON output type, to output file, is cut-off to 8k file. 

### ISSUE THREAD

**Issue by marked23** [tabulapdf/tabula-java#68]
When using the JSON output type, the output file is cut-short at 8192 bytes.  Interestingly, the CSV output does not have the same problem.

I can work-around the issue by not specifying an output file and instead redirect stdout to a file.  

Repro Info:
I am processing this pdf:
http://epra.ca/wp-content/uploads/2016/04/April-2016-National-List-v5.pdf

using this command line:
C:\tabula>java -jar ./tabula-java.jar -p2 -fJSON -oout.json April-2016-National-List-v5.pdf

The out.json file is exactly 8192 bytes (should be about 17k), and the end of the file leaves the json object unclosed.  It looks like this:
```
        {
          "top": 262.71,
          "left": 645.96,
          "width": 88.13997650146484,
          "height": 7.019999980926514,
          "text": "$0.75 ?"
        }
      ],
      [ { "top":<eof>
```

Thank you.

**Comment by jeremybmerrill** [comment]
Odd... I get a file that's 16384 bytes, but it's also truncated. I can confirm this, but don't know enough about Java to take it on. Mike or Manuel?

**Comment by jazzido** [comment]
Hrm...looks like we need to `flush()` the output stream in `CommandLineApp`.

**Comment by jazzido** [comment]
Fixed with @melisabok's contribution: 653599f3dee2e7dbe1939b465e5423d331a7c9d5
